app/containers/Dashboard/index.js

```diff
@@ -218,6 +218,9 @@ class DashboardRoot extends React.Component {
     this.power.downtime.call();
     this.power.totalSupply.call();
     this.power.activeSupply.call();
+    if (typeof this.power.minimumPowerUpSizeBabz === 'function') {
```

Comment:
After this is merged, I will create an issue to remind us to remove this check after the new ABI contracts are deployed successfully. (Lines 221 and 397)